visualizer/semanticsEditor.js

```diff
@@ -360,6 +366,177 @@
   }
   ohmEditor.parseTree.addListener('cmdclick:traceElement', toggleSemanticsEditor);
 
+  // Retrieve arguments from the menu entry's submenu
+  function retrieveArgumentsFromSubList(entry) {
+    var argumentWrappers = entry.querySelectorAll('ul li');
+    if (!argumentWrappers) {
+      return undefined;
```

Comment:
Then style checking will throw an error said that the other `return` statement in this method shouldn't return anything. i.e. `383:4  error  Expected no return value  consistent-return`